scene-format.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package turbocow;
+
+message Scene {
+  message Object {
```

Comment:
What about a sphere, a cube, and other primitives? They are not references to obj file and also have their own parameters (radius for sphere, size for cube)